scripts/run_all_tests.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up"
+
+mkdir bin
+GOBIN="`pwd`/bin"
+KUBECONFIG="`pwd`/config"
```

Comment:
can we check if kubeconfig exists or not? and error out if it isnt?

Reply:
No, the idea is to provide isolation. This would be the `global kubeconfig` for the test

Reply:
Using system level kubeconfig could conflict when multiple runs set different contexts